examples/pyfunc-upi/Pyfunc.ipynb

```diff
@@ -0,0 +1,667 @@
+{
```

Comment:
You'll need to stop the habit of including unrelated change into a PR. It should be a separate one.